source/extensions/common/matcher/BUILD

```diff
@@ -20,7 +21,7 @@ envoy_cc_library(
     ],
 )
 
-envoy_cc_library(
+envoy_cc_extension(
```

Comment:
Do you really need to change this just for docs?

Reply:
It's really an extension. It's main difference is that its proto is in cncf/xds.

Reply:
I wonder if some of these extensions in common really should have been in extensions/, but might be a bit late for that

Reply:
These are core extensions probably so it's a bit blurry.